webpack.config.js

```diff
@@ -9,19 +9,75 @@ var extractCss = new ExtractTextPlugin('[name].[chunkhash].bundle.css');
 
 module.exports = {
   devtool: 'source-map',
-  entry: {},
+  //entry: {},
```

Comment:
This change breaks the build. I'm not sure of it's purpose but I will investigate and revert it